frontend/src/lib-customizations/espoo/citizen/assets/i18n/sv.tsx

```diff
@@ -138,6 +138,17 @@ const sv: Translations = {
     types: {
       MESSAGE: 'Meddelande',
       BULLETIN: 'Bulletin'
+    },
+    messageEditor: {
+      newMessage: 'Nytt Meddelande',
+      receivers: 'Mottagare',
+      title: 'Titel',
+      message: 'Meddelande',
+      deleteDraft: 'Ta bort utkast',
+      send: 'Skicka',
+      search: 'Sök',
+      noResults: 'Inga resultat',
+      messageSendError: 'Misslyckades med att skicka meddelande'
```

Comment:
So have these been checked with someone who knows the language?

Reply:
Usually translations have been done by someone more knowledgeable only when there is a lot of text. For example, when the new application editor was built.